installation/single_machine_docker.rst

```diff
@@ -7,15 +7,23 @@ This section describes setting up a Citus cluster on a single machine using dock
 
 **1. Install docker and docker-compose**
 
-Follow the installation instructions for `Docker Engine <https://docs.docker.com/engine/installation/>`_ and `Docker Compose <https://docs.docker.com/compose/install/>`_ on your platform.
+* Install the `Docker Platform <https://www.docker.com/products/overview#/install_the_platform>`_.
+* (Linux only) On most platforms the previous step will install docker-compose as well as Docker, but on Linux you'll need to install docker-compose manually by downloading the binary and putting it in your path:
+
+  .. code-block:: bash
+
+    # Linux ONLY
+
+    curl -L https://github.com/docker/compose/releases/download/1.11.1/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
+    chmod +x /usr/local/bin/docker-compose
 
 **2. Start the Citus Cluster**
 
 Citus uses docker-compose to run and connect containers holding the database master node, workers, and a persistent data volume. To create a local cluster download our docker-compose configuration file and run it
 
 .. code-block:: bash
 
-  wget https://raw.githubusercontent.com/citusdata/docker/master/docker-compose.yml
+  curl -L https://raw.githubusercontent.com/citusdata/docker/master/docker-compose.yml > docker-compose.yml
   docker-compose -p citus up -d
```

Comment:
Super minor unrelated point for the note below (line 41).

If you have PostgreSQL running on your machine -> If you already have PostgreSQL running on your machine